caput/truncate.pyx

```diff
@@ -0,0 +1,161 @@
+"""Routines for truncating data to a specified precision."""
```

Comment:
Where is this used?

Reply:
Both in draco (in the future) and in driftscan (currently).